### ISSUE TITLE
docfx: remove URL fragments from Python TOC

### ISSUE THREAD

**Issue by tbpg** [googleapis/doc-templates#19]
## Expected Behavior
The TOC entries should not have URL fragments (`/path/to/doc#url-fragment`). They mess up the TOC parsing.

## Actual Behavior
Entries include URL fragments:

```
    section:
    - title: Overview
      path: /python/docs/reference/texttospeech/latest/google.cloud.texttospeech_v1.services.text_to_speech.html#google_cloud_texttospeech_v1_services_text_to_speech
    - title: google.cloud.texttospeech_v1.services.text_to_speech.TextToSpeechAsyncClient
      path: /python/docs/reference/texttospeech/latest/google.cloud.texttospeech_v1.services.text_to_speech.TextToSpeechAsyncClient.html#google_cloud_texttospeech_v1_services_text_to_speech_TextToSpeechAsyncClient
    - title: google.cloud.texttospeech_v1.services.text_to_speech.TextToSpeechClient
      path: /python/docs/reference/texttospeech/latest/google.cloud.texttospeech_v1.services.text_to_speech.TextToSpeechClient.html#google_cloud_texttospeech_v1_services_text_to_speech_TextToSpeechClient
```

## Steps to Reproduce the Problem

1. Run the tests for this repo.
1. Look at `testdata/python-small/site/api/toc.html`.

cc @tmatsuo

**Comment by dandhlee** [comment]
https://dotnet.github.io/docfx/tutorial/intro_toc.html#href-in-detail
looks like not much we can do from docfx side, will have to look into uri fragments being generated only for Python (href/topicHref both generate same results for Python with URI fragmentation)

**Comment by dandhlee** [comment]
Closed via #193
